dask/blockwise.py

```diff
@@ -763,110 +766,8 @@ def make_blockwise_graph(
     output_blocks=None,
     dims=None,
     io_deps=None,
+    keys=None,
 ):
-    """Tensor operation
```

Comment:
- [ ] consider putting doc string back. This is now an internal function so I want to only put it back if it helps developers